packages/components/src/tab-panel/test/index.tsx

```diff
@@ -9,7 +9,12 @@ import userEvent from '@testing-library/user-event';
  */
 import TabPanel from '..';
 
-const setupUser = () => userEvent.setup();
+jest.useFakeTimers();
```

Comment:
We've been recently working across the codebase to refactor unit tests _away_ from using take timers and `act()` call.

That would mean removing the `jest.useFakeTimers();` call and the `advanceTimers: jest.advanceTimersByTime,` option when setting `userEvent` up.

cc @tyxla

Reply:
Yes, we use fake timers only when we really have to. That means removing `jest.useFakeTimers()` and calling `const user = userEvent.setup();` without the `advanceTimers` setting.

Here are some PRs where we refactored from fake to real timers if that will help:

* https://github.com/WordPress/gutenberg/pull/47318
* https://github.com/WordPress/gutenberg/pull/47218
* https://github.com/WordPress/gutenberg/pull/47144
* https://github.com/WordPress/gutenberg/pull/47056
* https://github.com/WordPress/gutenberg/pull/46714